docs/60_initialize_vendors.md

```diff
@@ -0,0 +1,479 @@
+# Initialize Vendor Accounts
+
+The DANDI ecosystem relies on vendor services to operate.  So first you will need to set up accounts with the following vendors:
+
+• **Heroku**
+
+Provisions the servers and worker processes for the API. Heroku also handles the Postgres instance responsible
+for data models in the DANDI Archive.
+
+• **AWS**
+
+Provides storage buckets, as well as domain management, for resources across the DANDI ecosystem 
+
+• **GitHub**
+
+Serves as the authentication provider for accounts across the DANDI ecosystem
+
+• **Terraform Cloud**
+
+Manages provisioned resources across cloud vendors in a version-controlled manner. 
+
+• **Netlify**
+
+Deploys production frontend build, as well as s staging previews to assist with frontend development
+
+• **Sentry**
+
+Provides observability and monitoring for API events
+
+• **PyPI**
+
+Serves as a public registry for the `dandi-cli` tool
+
+Some services are not yet integrated within the main infrastructure:
+
+• https://github.com/dandi/backups2datalad - to populate/update https://github.com/dandi/dandisets, https://github.com/dandisets, and https://github.com/dandizarrs/
+
+• TODO: heroku logs
+
+• TODO: aws s3 access stats dump
+
+• TODO: con/tinuous dumps of CI logs 
+
+• TODO: zarr manifests generation (ATM not on drogon even)
+
+• TODO: access stats analysis/plots (yet to be finished/cron deployed)
+
+## Heroku
+
+##### Create your own Heroku account
+
+No special steps here, just create!
+
+##### Create a "Team" 
+
+Send invites to the appropriate e-mail addresses to onboard your team
+
+##### Create an "App"
+
+<br/><br/>
+<img
+src="../img/heroku_create_app.png"
+alt="heroku_create_app"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+You can name whatever you wish -- no need to add a pipeline, etc. **Be sure to note the app name**
+
+<br/><br/>
+<img
+src="../img/heroku_app_name.png"
+alt="heroku_app_name"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+Your app will be an empty template, but that is OK -- more to come here!
+
+##### Obtain Heroku API Key
+
+You'll need to provide access to Heroku for GitHub, Terraform, etc. -- thus, you'll need to generate an API Key
+
+First go to your `Account Settings`:
+
+<br/><br/>
+<img
```

Comment:
This image was a bit mis-leading when following the instructions. The image led us to believe we needed to go to our Heroku App -> Settings, however what we ended up needing to do was the following:
- Click the Profile icon in the top right
- Select "Account Settings" in the dropdown
- Scroll to find the "API Key" section